hera_pspec/pspecdata.py

```diff
@@ -774,6 +779,25 @@ def R(self, key):
             elif self.data_weighting == 'iC':
                 self._R[Rkey] = sqrtT.T * sqrtY.T * self.iC(key) * sqrtY * sqrtT
 
+            elif self.data_weighting == 'sinc_downweight':
+                r_param_key = (self.data_weighting,) + key
+                if not r_param_key in self.r_params:
+                    raise_warning("Warnging: no filter params specified for "
+                                    "clean weights! Defaulting to Identity!")
```

Comment:
warnging typo, also lets say "sinc weights" here

Reply:
I've removed the typo and changed "clean" to "since" :).